label_studio/io_storages/all_api.py

```diff
@@ -93,7 +94,10 @@ def get(self, request, **kwargs):
 class AllImportStorageListAPI(generics.ListAPIView):
 
     parser_classes = (JSONParser, FormParser, MultiPartParser)
-    permission_required = all_permissions.projects_change
+    if flag_set('fflag_feat_all_optic_478_access_of_cloud_storage_connectors_short'):
```

Comment:
This is evaluated at app load so it won't work with user/org ff targeting

Reply:
also there is no problem with this AllImportStorageListAPI::GET, we need to limit only POST/PATCH operations. I would say it might be dangerous to remove GET, probably we use storage info in some labeling processes for managers/reviewers/annotators.

Reply:
I don't think this class is the correct one at all.